### DISCUSSION TITLE
Need abilty to disable javascript on some sites

### DISCUSSION THREAD

**Discussion by deirdrew** [Floorp-Projects/Floorp#1147] [Feature Request]
Is this planned? I want to choose if to disable javascript on certain websites (paywalls, intrusive scripts, etc) and am surprised there is no facility for doing so :(

**Comment by regularentropy** [comment]
Hello. You can achieve this behavior by using Ublock Origin in advanced mode. It has an option to block JS on the sites you want.

**Reply by deirdrew** [reply]
Thanks, I actually installed noscript for now,  but I MUCH prefer it being a built-in setting you access simply be clicking on the "padlock". Next time at the URL, you see the red X so you know it's blocked and simply click the red X if you want to re-enable it.
![Image14](https://github.com/Floorp-Projects/Floorp/assets/56893921/6b93c823-d909-4dfe-b3c2-068b21a87bf8)
